src/templates/home/header.jsx

```diff
@@ -24,48 +24,88 @@ export const HomeHeader = () => {
+                fill="white"
+              />
+              <path
+                d="M25.2208 73.1536C25.9502 73.1095 28.5378 72.9746 30.182 72.7085C33.9827 72.1234 37.7172 71.3322 41.604 70.9186C42.2096 70.8499 43.196 70.5693 43.8135 72.6049C44.4534 74.7221 44.1283 76.1676 43.0245 76.4121C40.3395 76.9738 37.5826 77.0966 35.0095 77.9592C31.2132 79.2358 27.2063 79.3896 23.2689 79.6902C21.4867 79.8088 19.7142 79.8564 17.9284 79.8199C17.038 79.8107 16.1491 79.7719 15.2562 79.7536C14.3442 79.7015 13.534 79.6996 12.5098 79.6296C11.6044 79.566 10.6089 79.4588 9.48026 79.115C8.91469 78.9385 8.32015 78.692 7.73197 78.3266C7.14628 77.9703 6.5504 77.4704 6.07411 76.9085C4.98505 75.6073 4.5459 74.079 4.38872 72.9025C4.21909 71.6806 4.26032 70.6936 4.3042 69.8229C4.47726 66.8288 5.42203 63.9447 6.23823 61.1252C6.42605 60.4589 6.23633 59.5548 6.46613 58.8282C6.65076 58.2214 6.96107 57.219 7.31827 56.6721C7.47712 56.4334 7.65903 56.2078 7.66185 55.9338C7.70339 54.6637 8.53352 54.6309 10.6992 55.8023C11.18 56.0606 11.8701 56.1931 12.1298 56.5706C12.1439 56.5862 12.163 56.62 12.177 56.6357C13.3056 57.8674 12.4235 58.4218 12.2252 59.2276C11.2723 63.1482 10.6694 67.2068 10.6194 71.1819C10.5908 72.3216 10.8017 73.054 10.943 73.142C11.0248 73.2269 11.1223 73.2977 11.4031 73.3962C11.6813 73.4857 12.0776 73.5427 12.5661 73.5452C13.105 73.5533 13.7217 73.2767 14.4276 73.5024L14.416 73.4958C15.1479 73.4607 15.8831 73.3662 16.6002 73.3839C19.3442 73.4988 22.2031 73.2505 24.8422 73.2673C24.9313 73.2721 25.0682 73.1663 25.2208 73.1536Z"
```

Comment:
@Demoivre02 I believe you'd want to save this svg as an asset and import it here instead